packages/@aws-cdk/aws-apigatewayv2/lib/http/stage.ts

```diff
@@ -176,4 +176,18 @@ export class HttpStage extends HttpStageBase {
     const urlPath = this.stageName === DEFAULT_STAGE_NAME ? '' : this.stageName;
     return `https://${this.api.apiId}.execute-api.${s.region}.${s.urlSuffix}/${urlPath}`;
   }
+
+  /**
+   * The custom domain URL to this stage
+   */
+  public get customDomainUrl(): string {
```

Comment:
What do you think of calling this just `domainUrl()`?

Reply:
This reads cleaner. Updated